provisioning_client/devdoc/azure_provisioning_faq.md

```diff
@@ -0,0 +1,39 @@
+# Azure Iot Provisioning Client FAQ
+
+Here is where we shall try to answer question pertaining to the IoT Provisioning Client.  Please read the following documents for an overview of the DPS service.
+
+## x509 Provisioning
+
+- How does Provisining match the x509 certificates to an enrollment?
+
+  - The Provisioning Client add the x509 certificate (more on the certificate in a second) on the TLS stream.  The DPS service uses this certificate to look up the proper enrollment record.
+
+- Where does the x509 certificate comes from?
+
+  - This question depends on the situation:
```

Comment:
Make sure you link to: https://docs.microsoft.com/en-us/azure/iot-dps/concepts-security#x509-certificates